<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Review</class>
 <widget class="QDialog" name="Review">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tổng quan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QColumnView" name="columnView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>611</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Số ngày đã bán</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Số hóa đơn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tổng doanh thu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="ngayText">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="soHDText">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="doanhThuText">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>581</width>
       <height>341</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>139</number>
     </attribute>
     <column>
      <property name="text">
       <string>Tên</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Số hóa đơn</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tổng tiền</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Điểm tích lũy</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="statusButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tình hình bán hàng</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Khách hàng</string>
   </property>
  </widget>
  <zorder>columnView</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label</zorder>
  <zorder>statusButton</zorder>
  <zorder>clientButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
